azext_iot/digitaltwins/_help.py

```diff
@@ -140,6 +140,112 @@ def load_digitaltwins_help():
             az dt reset -n {instance_name}
     """
 
+    helps["dt data-history"] = """
+        type: group
+        short-summary: Manage and configure data history connections.
+    """
+
+    helps["dt data-history create"] = """
+        type: group
+        short-summary: Creates a data history connection between a Digital Twins instance and another resource.
+    """
+
+    helps["dt data-history create adx"] = """
+        type: command
+        short-summary: Creates a data-history connection between a Digital Twins instance and an Azure Database Explorer.
```

Comment:
We should be consistent with `data-history` verbiage in descriptions, in some case its 'data-history' and others 'data history'. I think the latter is best.